tastydoc/documentation/dotty/dotty.md

```diff
@@ -0,0 +1,5 @@
+# Package dotty
```

Comment:
Is this file (and the others in the `tastydoc/documentation/`) generated?

Reply:
Well spotted, Julien! It does look like this entire dir should be purged and added to `.gitignore`.